Mathlib/Algebra/Order/UpperLower.lean

```diff
@@ -42,15 +42,12 @@ section OrderedCommGroup
 variable {α : Type*} [OrderedCommGroup α] {s t : Set α} {a : α}
 
 @[to_additive]
-theorem IsUpperSet.smul (hs : IsUpperSet s) : IsUpperSet (a • s) := by
-  rintro _ y hxy ⟨x, hx, rfl⟩
-  exact mem_smul_set_iff_inv_smul_mem.2 (hs (le_inv_mul_iff_mul_le.2 hxy) hx)
+theorem IsUpperSet.smul (hs : IsUpperSet s) : IsUpperSet (a • s) := hs.image $ OrderIso.mulLeft _
```

Comment:
The style guide say `<|` should be used (which is what mathport wrote anyway)

> Note: while `$` is a synonym for `<|`, its
use in mathlib is disallowed in favor of `<|` for consistency as well as 
because of the symmetry with `|>`